nautilus_core/model/src/events/account/state.rs

```diff
@@ -93,7 +93,13 @@ impl PartialEq for AccountState {
     fn eq(&self, other: &Self) -> bool {
```

Comment:
As above, we only want to compare just the ID (like orders and other domain aggregates which contain state), as there are operations which just need to fetch the correct account and not further compare its state.